src/TestSuite/Fixture/FixtureDataManager.php

```diff
@@ -205,11 +206,18 @@ protected function runOperation(array $fixtures, callable $operation): void
         $dbs = $this->fixtureConnections($fixtures);
         foreach ($dbs as $connection => $fixtures) {
             $db = ConnectionManager::get($connection);
-            $db->transactional(function (ConnectionInterface $db) use ($fixtures, $operation): void {
+            if ($db->getDriver() instanceof Postgres) {
+                // disabling foreign key constraints is only valid in a transaction
+                $db->transactional(function (ConnectionInterface $db) use ($fixtures, $operation): void {
```

Comment:
Does the transaction make other databases behave incorrectly?

Reply:
It breaks sqlite because disabling foreign keys within a transaction is ignored. The transaction isn't really necessary. A failure breaks the test either way.